src/Handler/TagHandlerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace EzSystems\PlatformHttpCacheBundle\Handler;
+
+use Symfony\Component\HttpFoundation\Response;
+
+interface TagHandlerInterface
```

Comment:
Alternative name, which also overlaps with FOS, but with 2.x, is: `ReponseTaggerInterface`
see: http://foshttpcachebundle.readthedocs.io/en/latest/features/tagging.html

Similar to what they do: https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/master/src/Http/SymfonyResponseTagger.php

Reply:
Yes, I am aware of that. If there is consensus that ResponeTaggerInterface is a better name, I can change it

Reply:
It's fine keeping it like it is for now.
Can you rebase so we can merge?

Reply:
Rebased....